### ISSUE TITLE
Adding a tag collection

### ISSUE THREAD

**Issue by jeremiak** [18F/about_yml#10]
For the app store concept that @melodykramer and I are working on, we want to be able to assign tags to projects (i.e. "writing tool", "utility"). We want to add a key called `tags` that will be something like this:

```
"tags": {
    "type": "array",
    "description": "Tags that describe aspects of the project",
    "items": {"type": "string"},
    "uniqueItems": true
},
```

Thoughts @mbland @gboone ?

**Comment by mbland** [comment]
Sounds good to me!

**Comment by gboone** [comment]
Yeah, that sounds great. It'd help us figure out how to connect projects in the Dashboard to the blog and vice-versa. Tags we use on the blog: https://18f.gsa.gov/tags

**Comment by mbland** [comment]
Closed by #16.
